src/accelerate/commands/launch.py

```diff
@@ -257,6 +257,8 @@ def simple_launcher(args):
 
     current_env = os.environ.copy()
     current_env["USE_CPU"] = str(args.cpu)
+    current_env["MASTER_ADDR"] = args.main_process_ip
+    current_env["MASTER_PORT"] = str(args.main_process_port)
```

Comment:
Should this be under and `if args.num_machines > 1`? It seems related to multinode training.

Reply:
thanks for the suggestion, a patch is uploaded to fix it